content/en/tracing/trace_collection/custom_instrumentation/ios/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: iOS
+kind: documentation
+type: multi-code-lang
+external_redirect: /tracing/trace_collection/custom_instrumentation/ios/otel
```

Comment:
if we created this page to hold for all iOS tracing support shouldn't we also link here the old `Opentracing` docs ? Am I missing something ?

Reply:
for now we keep the OT and OpenTel support separate, I added a backlink from Opentracing to redirect customers to OpenTel.

I think we will do a bigger refactor at some point and move everything under the same umbrella of tracing but that's wishful thinking.